scripts/cmake/vcpkg_build_qmake.cmake

```
@@ -8,102 +8,104 @@ vcpkg_build_qmake()
+    )
+endfunction()
+    
 function(vcpkg_build_qmake)
     # parse parameters such that semicolons in options arguments to COMMAND don't get erased
-    cmake_parse_arguments(PARSE_ARGV 0 _csc "SKIP_MAKEFILES" "BUILD_LOGNAME" "TARGETS;RELEASE_TARGETS;DEBUG_TARGETS")
+    cmake_parse_arguments(PARSE_ARGV 0 arg
+        "SKIP_MAKEFILES"
+        "BUILD_LOGNAME"
+        "TARGETS;RELEASE_TARGETS;DEBUG_TARGETS"
+    )
 
     if(CMAKE_HOST_WIN32)
         if (VCPKG_QMAKE_USE_NMAKE)
-            find_program(NMAKE nmake)
-            set(INVOKE "${NMAKE}")
-            get_filename_component(NMAKE_EXE_PATH ${NMAKE} DIRECTORY)
-            set(PATH_GLOBAL "$ENV{PATH}")
+            find_program(nmake_executable nmake)
+            set(invoke_command "${nmake_executable}")
+            get_filename_component(NMAKE_EXE_PATH "${nmake_executable}" DIRECTORY)
             set(ENV{PATH} "$ENV{PATH};${NMAKE_EXE_PATH}")
             set(ENV{CL} "$ENV{CL} /MP${VCPKG_CONCURRENCY}")
         else()
             vcpkg_find_acquire_program(JOM)
-            set(INVOKE "${JOM}")
+            set(invoke_command "${JOM}")
         endif()
     else()
-        find_program(MAKE make)
-        set(INVOKE "${MAKE}")
+        find_program(make_executable make)
+        set(invoke_command "${make_executable}")
     endif()
 
     # Make sure that the linker finds the libraries used
-    set(ENV_PATH_BACKUP "$ENV{PATH}")
+    set(env_path_backup "$ENV{PATH}")
 
     file(TO_NATIVE_PATH "${CURRENT_INSTALLED_DIR}" NATIVE_INSTALLED_DIR)
 
-    if(NOT _csc_BUILD_LOGNAME)
-        set(_csc_BUILD_LOGNAME build)
+    if(NOT arg_BUILD_LOGNAME)
+        set(arg_BUILD_LOGNAME build)
     endif()
 
-    function(run_jom TARGETS LOG_PREFIX LOG_SUFFIX)
-        message(STATUS "Package ${LOG_PREFIX}-${TARGET_TRIPLET}-${LOG_SUFFIX}")
-        vcpkg_execute_build_process(
-            COMMAND ${INVOKE} -j ${VCPKG_CONCURRENCY} ${TARGETS}
-            NO_PARALLEL_COMMAND ${INVOKE} -j 1 ${TARGETS}
-            WORKING_DIRECTORY ${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-${LOG_SUFFIX}
-            LOGNAME package-${LOG_PREFIX}-${TARGET_TRIPLET}-${LOG_SUFFIX}
-        )
-    endfunction()
-
     # This fixes issues on machines with default codepages that are not ASCII compatible, such as some CJK encodings
-    set(ENV_CL_BACKUP "$ENV{_CL_}")
+    set(env_cl_backup "$ENV{_CL_}")
     set(ENV{_CL_} "/utf-8")
 
     #Replace with VCPKG variables if PR #7733 is merged
-    unset(BUILDTYPES)
+    vcpkg_list(SET buildtypes)
     if(NOT DEFINED VCPKG_BUILD_TYPE OR VCPKG_BUILD_TYPE STREQUAL "debug")
-        set(_buildname "DEBUG")
-        list(APPEND BUILDTYPES ${_buildname})
-        set(_short_name_${_buildname} "dbg")
-        set(_path_suffix_${_buildname} "/debug")
+        set(build_type_name "DEBUG")
+        vcpkg_list(APPEND buildtypes "${build_type_name}")
+        set(short_name_${build_type_name} "dbg")
+        set(path_suffix_${build_type_name} "/debug")
     endif()
```

```suggestion
    set(short_name_debug "dbg")
    set(path_suffix_debug "/debug")
    set(targets_debug "${arg_DEBUG_TARGETS}")
```

We can simplify this quite a bit (and same below)